### ISSUE TITLE
RBM : Number of epochs (9 gestures) [on cost]

### ISSUE THREAD

**Issue by FrancoisLasson** [FrancoisLasson/Temporal_DBN#2]
This issue is a response to #1 
_Context : 9 gestures (gesture 1 to 9 of CERV mocap database) and  4 instances for each one (a,b,c,d)_

Learning rate = 1e-3;  n_rbm_hidden = 30; batch_size =100, with shuffle.

In order to get the number of epochs necessaries to the RBM cost function stabilization during the training phase, we trained each RBMs during 10 000 epochs. 

The plot of this training is the following :
![rbm_1459943933 76-shuffle](https://cloud.githubusercontent.com/assets/11289891/14347231/49e39214-fcb6-11e5-942e-b0a00d5c6c68.png)

It seems to be stable from 8000 epochs.
